zk/conn.go

```diff
@@ -682,7 +726,11 @@ func (c *Conn) sendLoop() error {
 
 	for {
 		select {
-		case req := <-c.sendChan:
+		case req, ok := <-c.sendChan:
+			if !ok {
+				_ = c.conn.Close()
+				return nil
```

Comment:
Should this be `return errClosing`? It doesn't affect correctness, just the log lines.